src/pages/About.js

```diff
@@ -1,13 +1,39 @@
 import React, { Component } from 'react';
 import Toolbar from '../components/Toolbar.js';
+import { Container, Col, Row } from 'react-bootstrap';
+import '../css/About.css';
+import Members from '../components/Members.js';
 
 class About extends Component {
 	render() {
 		return (
-			<div> <Toolbar/> 
-			About
+			<Container>
+			<div className="div">
+			<Toolbar/>
+			<Row>
```

Comment:
Can you push all your content up a little higher to shorten the gap in between the toolbar and text.